web/css/style.css

```diff
@@ -0,0 +1,239 @@
+  font-size: 22px;
+}
+
+.section-subtitle {
+  font-size: 22px;
+}
+
+grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+
+grid > c {
+  /* overflow: hidden; */
+}
+
+grid > c[span="row"] {
+  /* grid-column-start: 1;
+  grid-column-end: -1; */
+  grid-column: 1 / -1;
+}
+
+c[height="30"] {
+  height: 30px;
+}
+
+.box {
+  background-color: #fafafa;
+  padding: 20px;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+}
+
+.description {
+  margin: 0;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 30px;
+  color: #808080;
+}
+
+.bAtention {
+  background-color: #FE004E;
+  color: #fff;
+  height: 40px;
+  padding: 0 24px;
+  border-radius: 20px;
+  border: 0;
+  font-size: 12px;
+  font-weight: bold;
+}
+
+.circle {
+  background-color: #2DD070;
+  color: #fff;
+  border-radius: 50%;
+  height: 40px;
+  width: 40px;
+  border: 0;
+}
+
+
+
+.tCentered {
+  text-align: center;
+}
+
+.avatar {
+  width: 100%;
+  border-radius: 5px;
+  border: 1px solid #808080;
+}
+
+.image-wrapper {
+  border-radius: 6px;
+  padding: 8px;
+  border: 1px solid #808080;
+}
+
+
+c[span="1"] {
+  grid-column-end: span 1;
+}
+
+a[class^="bDisplay__"] {
+  width: 100%;
+  padding-top: 100%;
+  border-radius: 100%;
+  display: block;
+  position: relative;
+}
+
+a[class^="bDisplay__"] > [class^="my-cv-"] {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 8vw;
+}
+
+.bDisplay__onahau {
+  background-color: #CCE4FC;
+}
+.bDisplay__grannyApple {
+  background-color: #D4F5E2;
+}
+.bDisplay__oasis {
+  background-color: #FDEBCF;
+}
+.bDisplay__blueChalk {
+  background-color: #F1DEFF;
+}
+.bDisplay__pastelPink {
+  background-color: #FFCCDB;
+}
+.bDisplay__azalea {
+  background-color: #FADBD8;
+}
+.bDisplay__moonRaker {
+  background-color: #DFDDF9;
+}
+.bDisplay__whiteIce {
+  background-color: #D4F6F1;
+}
+.bDisplay__janna {
+  background-color: #F4E5D7;
+}
+.bDisplay__doublePearlLusta {
+  background-color: #FCF3CF;
+}
+
+.skills__torch-red {
+  background-color: #FD004D;
+}
+.skills__shamrock {
+  background-color: #2DD070;
+}
+.skills__buttercup {
+  background-color: #F29E1A;
+}
+
+.skills__torch-red,
+.skills__shamrock,
+.skills__buttercup {
+  height: 40px;
+  padding: 0 24px;
+  border-radius: 20px;
+  font-size: 12px;
+  color: white;
+  font-weight: bold;
+  margin-right: 16px;
+  margin-bottom: 16px;
+}
+
+.skills-list {
+  margin: 20px 0;
+}
+
+.skills-legend {
+  border: 2px solid #808080;
+  list-style-type: none;
+  padding-left: 0;
+  border-radius: 15px;
+  overflow: hidden;
+}
+
+.skills-legend_item:not(:last-child) {
+  border-bottom: 2px solid #808080;
+}
+
+.skills-legend_item:hover {
+  background-color:#fafafa;
+}
+
+.skills-legend_item > span[class^="my-cv-"] {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  height: 30px;
+  /* font-size: 6vw; */
```

Comment:
once you're happy with your code, remove unnecessary comments

Reply:
done